docs/topics/kmm/kmm-getting-started.md

```diff
@@ -1,22 +1,54 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: cross platform mobile development, android vs ios development)
 
-## Get familiar with KMM
+> Kotlin Multiplatform Mobile is in [Alpha](components-stability.md). Language features and tooling may change in future
+> Kotlin versions.
+> 
+> Beta is expected in spring 2022. See [KMM Beta Roadmap](https://youtu.be/LE-Dy9XCLxk) video
+> to learn about upcoming features.
+>
+{type="note"}
+
+Kotlin Multiplatform Mobile (KMM) is an SDK designed to simplify development of cross-platform mobile applications.
+You can share common code between iOS and Android apps and write platform-specific code only where it's necessary.
+For example, to implement a native UI or when working with platform-specific APIs.
 
 Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Developer Advocate Ekaterina 
-Petrova will explain what Kotlin Multiplatform Mobile is and how you can use it in your project. Together with Ekaterina 
-you'll set up an environment and prepare for creating your first cross-platform mobile application with KMM.
+Petrova explains what Kotlin Multiplatform Mobile is and how you can use it in your project. With Ekaterina,
+you'll set up an environment and prepare to create your first cross-platform mobile application:
 
 <video href="mdN6P6RI__k" title="Kotlin Multiplatform Multiverse, Episode 1: Meet KMM!"/>
 
-## Start KMM from scratch
+You can also check out other videos about [Kotlin Multiplatform Multiverse](https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C) on YouTube.
+
+## Supported platforms
```

Comment:
I thought that we agreed to move supported platforms to our general file with platforms and include a link here. I am not sure that such detailed description is required for Get started. We could make it high level and move all details to the platform reference.

Reply:
Trimmed platform descriptions following Kate's suggestions